plugins/catalog-backend-module-gitlab/src/lib/client.ts

```diff
@@ -63,6 +65,40 @@ export class GitLabClient {
     return this.pagedRequest(`/projects`, options);
   }
 
+  async listUsers(options?: ListOptions): Promise<PagedResponse<GitLabUser>> {
+    return this.pagedRequest(`/users`, options);
+  }
+
+  async listGroups(options?: ListOptions): Promise<PagedResponse<GitLabGroup>> {
+    return this.pagedRequest(`/groups`, options);
+  }
```

Comment:
Could these two methods respect the `group` option (i.e. scope your discovery of users and groups to a single group and its subgroups) like `listProjects` does?

https://github.com/backstage/backstage/blob/38c32326456b5dcef0c16b5ff011cf4181a94d7f/plugins/catalog-backend-module-gitlab/src/lib/client.ts#L53-L61